.env.example

```diff
@@ -20,3 +20,9 @@ LADI_THRESHOLD=1.5 # float tunes the anomaly threshold based on the max value of
 
 LOCAL_DATA_FILE=validation_data/Hadoop_2k.json
 LOCAL_RESULTS_FILE=/tmp/local-results.txt
+CEPH_KEY=<REPLACE_WITH_S3_KEY> # The s3 key
```

Comment:
What are these for? Are they used somewhere in the code?

Also, I would suggest using some more generic naming - `S3_...` intead of `CEPH_` probably - as we care about S3 API rather than it being specifically Ceph

Reply:
Good point. S3 is better naming convention. I'll change that.

Reply:
@vpavlin Its fixed my friend!